<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertMnist</class>
 <widget class="QWidget" name="ConvertMnist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据格式转换-Mnist</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>结果名:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>图片集:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>标签集:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>结果地址:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_ResultName"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_MnistResult"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_MnistlabelSet"/>
   </item>
   <item row="5" column="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QRadioButton" name="radioButton_Lmdb">
     <property name="text">
      <string>Lmdb</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QRadioButton" name="radioButton_LevelDb">
     <property name="text">
      <string>Leveldb</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_Convert">
     <property name="text">
      <string>转换</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButton_Browser3">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_Browser2">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton_Browser1">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_MnistImageSet"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>进度:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Browser1</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMnist</receiver>
   <slot>Browser1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser2</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMnist</receiver>
   <slot>Browser2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browser3</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMnist</receiver>
   <slot>Browser3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Convert</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMnist</receiver>
   <slot>DoConvert_Mnist_Data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Browser1()</slot>
  <slot>Browser2()</slot>
  <slot>Browser3()</slot>
  <slot>DoConvert_Mnist_Data()</slot>
 </slots>
</ui>
